IFComp/root/src/_current_entry_row.tt

```diff
@@ -11,6 +11,10 @@
         [% FILTER html_para %]
         	[% entry.blurb | html %]
         [% END %]
+        
+        [% IF entry.warning %]
+            <p><strong>Content warning:</strong> [% entry.warning %]</p>
```

Comment:
You should probably put an `html` filter on this, to prevent arbitrary markup/javascript from being inserted into the page.